File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
        bool fst = true;
    for (auto q : s)
    {
        if (q == ' ')
        {
            answer += q;
            fst = true;
            continue;
        }

        if (fst)
        {
            fst = false;
            if (q <= 'z' && q >= 'a')
                answer += (q - 32);
            else answer += q;
        }
        else
        {
           if (q <= 'Z' && q >= 'A')
                answer += (q + 32);
           else answer += q;
        }
    }
    return answer;
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cs
using System;
public class Solution {
    public string solution(string s) {
        s = s.ToLower();
        string answer = "";
        for(int i = 0; i<s.Length;i++)
        {
            if (i == 0 || s[i - 1] == ' ') answer += char.ToUpper(s[i]);
            else answer += s[i];
        }
        
        return answer;
    }
}